### PR TITLE
Drop trailing slash from api get and delete args

### PR DESCRIPTION
**Pull request by soulshake**
Dumbest, tiniest PR ever, but i keep typoing it and it bugs me. (Feel free to reject.)

Before:

```
$ /usr/local/bin/convox api get /apps/
ERROR: response status: 404 404 page not found
```

After:

```
$ convox api get /apps/
[
  {
    "name": "cv-soulshake-net",
    "release": "RRANZYKRPSM",
    "status": "running"
  },
  {
    "name": "trainingwheels",
    "release": "",
    "status": "running"
  }
]
```

### CODE REVIEW

**Comment by soulshake** [comment]
[Document CF Events in app and rack logs](https://trello.com/c/RUDMOfxb/2070-document-cf-events-in-app-and-rack-logs)

**Comment by soulshake** [comment]
[Drop trailing slash from api get and delete args](https://trello.com/c/xxqMspxX/2159-drop-trailing-slash-from-api-get-and-delete-args)

File: cmd/convox/api_test.go [hunk 1]
```diff
@@ -275,3 +275,22 @@ func TestApiGetNoArg(t *testing.T) {
 		},
 	)
 }
+
+// TestApiGet404 should ensure an error is returned when a user runs 'convox api get' with an invalid API endpoint.
```
Comment (MiguelMoll) [inline cmd/convox/api_test.go]: Should read `TestApiTrailingSlash`

Comment (soulshake) [inline cmd/convox/api_test.go]: Whoops, thanks

**Pull request closed without merging**